src/client/lobby/lobby.js

```diff
@@ -47,6 +50,10 @@ export class Lobby extends Component {
     }
 };
 
+function redirectToRoom(arg){
+    browserHistory.push(`/room/${arg}`);
```

Comment:
Does this add the page to history as you is going to it? Isn't it supposed to be the previous page that's added?